### PR TITLE
Update issue template (thanks jart@) and add env collection script.

### PR DESCRIPTION
**Pull request by aselle**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
...
+- *TensorFlow version* (use command below):
 - *Bazel version (if compiling from source)*:
 - *CUDA/cuDNN version*:
 - *GPU Model and Memory*:
 - *Exact command to reproduce*:
 
+You can collect some of this information using our environment capture script https://github.com/tensorflow/tensorflow/blob/master/tools/
+You can collect the TensorFlow version with
+```sh
+python -c "import tensorflow as tf; print (tf.__git_version__, tf.__version__)"
```
Comment (drpngx) [inline ISSUE_TEMPLATE.md]: Could we use GIT_VERSION and VERSION instead?

File: tools/tf_env_collect.sh [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+echo >> $OUTPUT_FILE
+echo '== check for virtualenv =========================================' >> $OUTPUT_FILE
+python -c "import sys;print(hasattr(sys, \"real_prefix\"))" >> $OUTPUT_FILE
+
+echo >> $OUTPUT_FILE
+echo '== tensorflow import ============================================' >> $OUTPUT_FILE
+cat <<EOF > /tmp/check_tf.py
+import tensorflow as tf;
+print("tf.__version__ = %s" % tf.__version__)
```
Comment (drpngx) [inline tools/tf_env_collect.sh]: same here

File: tools/tf_env_collect.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
```
Comment (drpngx) [inline tools/tf_env_collect.sh]: bash -u

**Review by drpngx — CHANGES_REQUESTED** [review]

File: tools/tf_env_collect.sh (hunk 3 above)
Comment (aselle) [inline tools/tf_env_collect.sh]: That does not seem to work. What is it supposed to do?

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (aselle) [inline ISSUE_TEMPLATE.md]: ok

File: tools/tf_env_collect.sh (hunk 2 above)
Comment (aselle) [inline tools/tf_env_collect.sh]: ok

**Comment by drpngx** [comment]
It generates an error if a variable is undefined. That's good for spotting
typos. You can set -u in the script.

The style guide mandates -eu, but -e which causes the script to end on any
non success command is too harsh in that case

**Comment by aselle** [comment]
It seems I can do
```
#!/bin/bash -u
```
but I don't think I can use env (which gives robustness to bash not installed in /bin)
```
#!/usr/bin/env bash -u
```
doesn't work because the -u doesn't get sent to bash.

**Comment by drpngx** [comment]
OK. From the bash script you can just do set -u first thing.

**Comment by aselle** [comment]
@drpngx, your changes are now addressed I believe.

**Review by drpngx — APPROVED** [review]
Sorry I missed this

**Comment by drpngx** [comment]
Jenkins, test this please.

**Pull request merged**